docs/getting_started.rst

```diff
@@ -0,0 +1,166 @@
+Getting Started
+===============
+
+The following sections provide information on how to setup your Conda environment and install WaterTAP on your macOS, Linux, or Windows computer.
+
+Using Conda environments
+------------------------
+
+Conda environments are a way to create and manage multiple sets of packages and/or Python versions on the same system without incurring conflicts. Each Conda environment is a dedicated directory, separate from other Conda environments and the operating system's own directories, containing its own collection of packages, executables, and Python installation, including the Python interpreter. Once a Conda environment is *activated*, using the ``conda activate`` command in a terminal/console, the environment's own version of Python will be used to run commands or interactive sessions for the remainder of the session. For these reasons, Conda environments are especially useful to install and manage multiple projects (and/or multiple *versions* of the same project) on the same computer with minimal effort, as they provide a way to seamlessly switch between different projects without conflicts.
+
+Using Conda environments is not mandatory to be able to install and use WaterTAP; however, it is strongly recommended. To use Conda environments, the ``conda`` package manager is required. Refer to the `Conda installation guide <https://idaes-pse.readthedocs.io/en/stable/tutorials/getting_started/index.html#installation>`_ for detailed steps on how to install Conda for your operating system.
+
+.. _install:
+
+Install on macOS
```

Comment:
I think it makes more sense to start with Linux/Windows installation and let Mac instructions follow. My reasoning is that running on Mac is more finnicky whereas Linux/Windows is more of our strong suit for the time being.

Reply:
Moved the Linux/Windows installation section before the Mac section.